CONTRIBUTING.md

```diff
@@ -1,4 +1,6 @@
-# Node.js Community Contributing Guide 1.0
+# Node.js Community Committee Contributing Guide 1.1
+
+**First things first:** If you're unsure of anything, please _don't hesitate to ask a question in a discussion or create an issue_. Nobody will be upset with you; we'll always try to include you in the work you care about. The worst that can happen is that you'll be politely asked to change something. Any kind of contribution, no matter how big or small, helps!
```

Comment:
👍 LGTM
"The **worst** that can happen"
Not sure, if we can re-phrase this? 
There is nothing wrong but it just feels like something bad may happen.

Reply:
This comment got me thinking, but I'm undecided on making a suggestion. Thanks for raising!

Reply:
I think I'm going to merge this for now, but TOTALLY open to a PR to change the phrasing! 💥